.azure-pipelines/client.test.live.yml

```diff
@@ -28,21 +28,27 @@ jobs:
   - task: Npm@1
     inputs:
       command: 'custom'
-      customCommand: 'run install-client'
-    displayName: 'npm run install-client'
+      customCommand: 'run install-client-$(Package)'
```

Comment:
I assume this is a variable you specify at queue time? Could you add a comment about it?

Reply:
It's set in the pipeline UI (though technically it's not settable a queue time -- it's hardcoded into the pipeline for each package).  Will add comment similar to https://github.com/Azure/azure-sdk-for-net/blob/master/.azure-pipelines/client.yml#L1-L2:

```
# External variables:
# ProjectFile - The project to build and test. This variable is defined in pipeline web ui because we want to be able to provide it at queue time and that isn't supported in yaml yet.
```